src/main/msp/msp.c

```diff
@@ -1507,6 +1507,8 @@ case MSP_NAME:
         // Added in API version 1.43
         sbufWriteU8(dst, gpsConfig()->gps_set_home_point_once);
         sbufWriteU8(dst, gpsConfig()->gps_ublox_use_galileo);
+        // Added in API version 1.47
+        sbufWriteU8(dst, gpsData.platformVersion);
```

Comment:
THis is inconsistent with MSP_SET_GPS_CONFIG (and platformVersion is not configurable)